src/tx-builder/services/DydxGovernance.ts

```diff
@@ -68,6 +68,8 @@ import BaseService from './BaseService';
 import ERC20Service from './ERC20';
 import GovernanceDelegationTokenService from './GovernanceDelegationTokenService';
 
+const DEPLOYMENT_BLOCK: number = 12816310;
```

Comment:
Confusing process we copied from Aave, but we put all TxBuilder constants in this file: https://github.com/dydxfoundation/governance-contracts/blob/master/src/tx-builder/config/index.ts

Also naming nit: `DYDX_GOVERNOR_DEPLOYMENT_BLOCK`

Reply:
nit: use `DYDX_GOVERNOR_DEPLOYMENT_BLOCK` instead